### PR TITLE
New version: OnionShare.OnionShare version 2.6.1

### PR DESCRIPTION
**Pull request by Exorcism0666**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/141323)

### CODE REVIEW

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping Exorcism0666.
>[!warning]
>With the new version of Komac, numerous Pull Requests may contain bugs or unwanted elements. If such issues arise, please tag russellbanks so that these bugs can be addressed in future updates.

>[!important]
>Please carefully review these Pull Request before merging. If it is a Pull Request for removing incorrect content and the URLs are issue, free upon manual checking, please close this Pull Request directly. (It is best to inform Exorcism0666 of the closure.)

良い一日をお過ごしください！

**Comment by stephengillie** [comment]
Automatic Validation ended with:   Installation failed with exit code -1978334957  > 2024-02-28 10:55:07.230 [CLI ] Installer [X64,wix,User,en-US] not applicable: Machine is not compatible with X64  > 2024-02-28 10:55:19.430 [CLI ] ShellExecute installer failed: 1633 > MSI (s) (D0:48) [10:55:19:324]: Product: OnionShare -- Installation failed.   (Automated response - build 846.)

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Installation failed with exit code -1978334957
> 2024-02-28 10:55:19.430 [CLI ] ShellExecute installer failed: 1633
> MSI (s) (D0:48) [10:55:19:324]: Product: OnionShare -- Installation failed.

(Automated response - build 849.)

File: manifests/o/OnionShare/OnionShare/2.6.1/OnionShare.OnionShare.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+- silentWithProgress
+UpgradeBehavior: install
+ProductCode: '{5BD1DFBB-1C9E-4CC1-8E95-2D114B26FC5A}'
+ReleaseDate: 2024-02-28
+AppsAndFeaturesEntries:
+- UpgradeCode: '{ED7F9243-3528-4B4A-B85C-9943982E75EB}'
+Installers:
+- Architecture: x86
+  InstallerUrl: https://github.com/onionshare/onionshare/releases/download/v2.6.1/OnionShare-win64-2.6.1.msi
+  InstallerSha256: CC75DAE12ABE07452D6DE6B7A4028A3F843816AA24AB917826C6FB2AFE87A9C9
```
Comment (stephengillie) [inline manifests/o/OnionShare/OnionShare/2.6.1/OnionShare.OnionShare.installer.yaml:23]:
```suggestion
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
This appears to be a 64-bit application.

**Comment by stephengillie** [comment]
Hi @Exorcism0666,

> This PR's version number 2.6.1 has 3 parameters (sets of numbers between dots - major, minor, etc), which is greater than the current manifest's version 2.6, which has 2 parameters.

Does this PR's version number **exactly** match the version reported in the Apps & features Settings page? (Feel free to attach a screenshot.)

(Automated response - build 853)

**Comment by stephengillie** [comment]
Hi @Exorcism0666,

This manifest adds Apps and Features entries that aren't present in previous PR versions. These entries should be added to the previous versions, or removed from this version.

(Automated response - build 853.)

**Comment by Exorcism0666** [comment]
> Hi @Exorcism0666,
> 
> This manifest adds Apps and Features entries that aren't present in previous PR versions. These entries should be added to the previous versions, or removed from this version.
> 
> (Automated response - build 853.)

These entries should be added to the previous versions.

**Comment by stephengillie** [comment]
I'm not completely sure, but I believe I was in error about these specific Apps and Features entries being required in all manifests for the PackageIdentifier.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**